lib/active_fulfillment/models/dotcom_distribution/post_item.rb

```diff
@@ -118,7 +118,9 @@ def item_to_xml(xml)
           xml.send(:"description") do
             xml.cdata self.description
           end
-          xml.send(:"quantity", self.quantity, inject_nil(self.quantity))
+          if quantity
```

Comment:
`PurchaseOrder` uses `PostItem` to create a new purchase order for the specific item for a `quantity`. The catch is that PurchaseOrder requires item to have a quantity while PostItem will fail if quantity is present.

I think that the objects in the gem require some clean up as there's a bit of duplication going on. If anything, we could start thinking about removing ActiveModel and going with another approach but I think that's low priority.